docs/01_cva6_user/ip-xact/cva6_csr.md

```diff
@@ -1,7 +1,7 @@
 # REGISTERS CSR CV32A6
 
 ## Floating-Point Accrued Exceptions Register 
-### *AddressOffset*: 1 
+### *AddressOffset*: 'h1 
```

Comment:
Suggest we expand this to the full 12-bits: `12'h001`.

Reply:
Great idea! Yes we can modify the MarkDown to expand it to the full 12-bits.